metals/src/main/scala/scala/meta/internal/metals/BuildServerConnection.scala

```diff
@@ -327,17 +356,12 @@ object BuildServerConnection {
     // Block on the `build/initialize` request because it should respond instantly
     // and we want to fail fast if the connection is not
     val result =
-      try {
-        if (serverName == SbtBuildTool.name) {
-          initializeResult.get(60, TimeUnit.SECONDS)
-        } else {
-          initializeResult.get(20, TimeUnit.SECONDS)
-        }
-      } catch {
-        case e: TimeoutException =>
-          scribe.error("Timeout waiting for 'build/initialize' response")
-          throw e
+      if (serverName == SbtBuildTool.name) {
+        initializeResult.get(60, TimeUnit.SECONDS)
+      } else {
+        initializeResult.get(20, TimeUnit.SECONDS)
```

Comment:
This was 5 in the past as well before the sbt pr, so hopefully this being a bit longer will help as well.